myapp/spec/system/sessions_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe 'Sessions', type: :system do 
+  describe 'login page' do 
+    it 'should correctly render components' do 
+      visit login_path
+
+      # page title
+      expect(page).to have_content('Log in')
+      # username field
+      expect(page).to have_field('Username')
+      # password field
+      expect(page).to have_field('Password')
+      # login button
+      expect(page).to have_button('Log in')
+      # signup text
+      expect(page).to have_content('New user? Click ')
```

Comment:
may be we can remove the space after "click "??

Reply:
Removed it in 1928cf4